src/contractManagers/CommonContractManager.ts

```diff
@@ -0,0 +1,32 @@
+import { Contract } from '../ethLibAdapters/EthLibAdapter'
+import { Address } from '../utils/basicTypes'
+import { sentinelModules } from '../utils/constants'
+
+abstract class CommonContractManager {
```

Comment:
Maybe call this a `ModuleManager`? All of the functionality has to do with modules.